test/membrane/rtp/outbound_tracking_serializer_test.exs

```diff
@@ -0,0 +1,33 @@
+defmodule Membrane.RTP.OutboundTrackingSerializerTest do
+  use ExUnit.Case, async: true
+
+  alias Membrane.Buffer
+  alias Membrane.RTP.{OutboundTrackingSerializer, Packet}
+
+  test "generates padding packets" do
+    serializer_state = %OutboundTrackingSerializer.State{}
+
+    buffer = %Buffer{
+      metadata: %{
+        rtp: %{
+          is_padding?: true,
+          ssrc: 1,
+          payload_type: 0,
+          marker: true,
+          extensions: [],
+          csrcs: [],
+          timestamp: 0,
+          sequence_number: 0
+        }
+      },
+      payload: <<>>
+    }
+
+    assert {{:ok, buffer: {:output, %Buffer{payload: payload}}}, _state} =
+             OutboundTrackingSerializer.handle_process(:input, buffer, nil, serializer_state)
+
+    assert byte_size(payload) == 256
```

Comment:
We should check last byte against correct value - it should denote padding size

Reply:
In this case, yes. However, now that I think about it, we can calculate the size of the header in this particular example, so we can just verify the entire padding binary